data/core/units/monsters/Fire_Dragon.cfg

```diff
@@ -5,6 +5,7 @@
     race=monster
     # workaround to cut the transparent part when displayed in the help
     image="units/monsters/fire-dragon.png~CROP(3,0,154,154)"
+    profile="portraits/monsters/fire-dragon.png"
     [standing_anim]
         start_time=0
```

Comment:
Maybe let's move that 2 lines up, because the comment is about the cropping and the standing animation.